src/test/payload/UpdateAaveTokenPayload.t.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {Test} from 'forge-std/Test.sol';
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+import {ProxyHelpers} from 'aave-helpers/ProxyHelpers.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {AaveTokenV3} from '../../AaveTokenV3.sol';
+import {UpdateAaveTokenPayload} from '../../payload/UpdateAaveTokenPayload.sol';
+
+contract UpdateAaveTokenPayloadTest is Test {
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 17635720);
+  }
+
+  function testExecute() public {
+    AaveTokenV3 aaveToken = new AaveTokenV3();
+
+    UpdateAaveTokenPayload payload = new UpdateAaveTokenPayload(address(aaveToken));
+
+    GovHelpers.executePayload(vm, address(payload), AaveGovernanceV2.LONG_EXECUTOR);
+
+    address newImpl = ProxyHelpers.getInitializableAdminUpgradeabilityProxyImplementation(
+      vm,
+      AaveV3EthereumAssets.AAVE_UNDERLYING
+    );
+
+    assertEq(newImpl, address(aaveToken));
```

Comment:
would it make sense to add more tests, like base functionalities? we can still transfer, etc etc
?
that it still has correct name, decimals, symbol, etc, stuff that could get broken by initialization